spec/config.schema.json

```
@@ -1915,69 +1915,27 @@
                 "sampling": {
                   "type": "object",
                   "propertyNames": {
-                    "enum": ["type", "value", "server_url"]
+                    "enum": ["server_url", "trace_id_ratio"]
                   },
```

This whole key can be removed. Use `additionalProperties: false` instead.
```suggestion
                  "additionalProperties": false,
```

---

done, thx